db.go

```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/binary"
```

Comment:
This doesn't look gofmt formatted, please check!

Reply:
Formatted :checkered_flag: